projects/plugins/crm/src/js/data/query-functions.ts

```diff
@@ -7,24 +6,43 @@ declare let jpcrmAutomationsInitialState: any;
 
 const v4ApiRoot = `${ jpcrmAutomationsInitialState.apiRoot }jetpack-crm/v4`;
 
-const api = axios.create( {
-	baseURL: v4ApiRoot,
-	headers: { 'X-WP-Nonce': jpcrmAutomationsInitialState.apiNonce },
-} );
+type fetchAutomationsAPIOptions = {
+	headers: HeadersInit;
+	method: string;
+	body?: string;
+};
+
+const callAutomationsAPI = async ( relative_url: string, payload?: object ) => {
+	const options: fetchAutomationsAPIOptions = {
+		headers: {
+			'X-WP-Nonce': jpcrmAutomationsInitialState.apiNonce,
+			'Content-Type': 'application/json',
+		},
+		method: 'GET',
+	};
+	if ( payload ) {
+		options.method = 'POST';
+		options.body = JSON.stringify( payload );
+	}
+	const response = await fetch( v4ApiRoot + relative_url, options );
+	return await response.json();
```

Comment:
One thing I'm not sure about here, as we don't appear to have done that before (in this file anyway) is what happens if the response is an error or unexpected result. Perhaps not for this PR anyway :)

Reply:
Yeah, there are definitely improvements to be made...in the future!